docs/get-details/dapps/smart-contracts/apps/index.md

```diff
@@ -269,7 +272,7 @@ itxn_submit
 All inner transactions will be stored as inner transactions within the outer application transaction. These can be accessed by getting the transaction id as normal and looking for the `inner-txns` header in the response.
 
 ## Allowed transaction properties
-Since TEAL 6 all application types can be used within inner transactions. If you're using TEAL 5 you will only be able to make payment and asset transfer transactions, with some properties such as `RekeyTo` not being allowed.
+Since TEAL 6, all transaction types can be used within inner transactions. If you're using TEAL 5 you will only be able to make payment and asset transfer transactions, with some properties such as `RekeyTo` not being allowed.
```

Comment:
Reviewer note:  I changed _application_ to _transaction_ here because I thought it's a typo.  Can folks take a look?

Reply:
Indeed, it appears to be a typo.

Reply:
It is a typo

Reply:
Thanks for the review - resolving.